examples/OcclusionSaliency.ipynb

```diff
@@ -0,0 +1,211 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Failed to import psycopg2: No module named 'psycopg2'\n",
+      "Failed to import psycopg2: No module named 'psycopg2'\n",
+      "Failed to import psycopg2: No module named 'psycopg2'\n",
+      "Failed to import caffe module: No module named 'caffe'\n",
+      "Failed to import psycopg2: No module named 'psycopg2'\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2021-05-20 16:56:53--  https://raw.githubusercontent.com/pytorch/hub/master/imagenet_classes.txt\n",
+      "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.110.133, 185.199.111.133, 185.199.108.133, ...\n",
+      "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.110.133|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 10472 (10K) [text/plain]\n",
+      "Saving to: ‘imagenet_classes.txt.4’\n",
+      "\n",
+      "imagenet_classes.tx 100%[===================>]  10.23K  --.-KB/s    in 0s      \n",
+      "\n",
+      "2021-05-20 16:56:54 (48.0 MB/s) - ‘imagenet_classes.txt.4’ saved [10472/10472]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"\n",
+    "This notebook contains a working example show usage of the new\n",
+    "image classifier based saliency map API for choosing to use \n",
+    "a occlusion based saliency map algorithm.\n",
+    "\n",
+    "Reference for implementation: Zeiler MD, Fergus R. Visualizing \n",
+    "and understanding convolutional networks. InEuropean conference\n",
+    "on computer vision 2014 Sep 6 (pp. 818-833). Springer, Cham.\n",
+    "\n",
+    "\"\"\"\n",
+    "from xaitk_saliency.interfaces.perturb_image import PerturbImage\n",
+    "from xaitk_saliency.interfaces.vis_sal_classifier import ImageClassifierSaliencyMapGenerator\n",
+    "from smqtk_core.configuration import from_config_dict\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "from PIL import Image\n",
+    "from torchvision import transforms\n",
+    "from torch.autograd import Variable\n",
+    "import torchvision.models as models\n",
+    "from sklearn.linear_model import LogisticRegression\n",
+    "import torch\n",
+    "from sklearn.preprocessing import minmax_scale\n",
+    "import numpy as np\n",
+    "from operator import itemgetter\n",
+    "\n",
+    "# Test image to be explained\n",
+    "test_image = 'catdog.png'\n",
```

Comment:
The other images included here don't seem to be being used. Should they be removed? Or should more cells be added here to show results for all 3 images?

Reply:
Separately, what is the license for those images? Are we allowed to bundle them in this repo, or do they need to be downloaded by the user?

Reply:
Deleted the images not being used. 

The image was originally from the MS COCO dataset. Looks like a CCA license. Here are the [terms of use](https://cocodataset.org/#termsofuse). How do we generally handle images in the repo? I can grab a standard Lenna image if that is advised.

Reply:
It is generally preferred to not have binary content in the repo unless needed for reliability and consistency reasons (e.g. small data files for tests, or reproducible examples). Lenna image is not OK to redistribute (e.g. [see here](https://github.com/matplotlib/matplotlib/issues/4927)). COCO is fine but it's license (CC by 4.0) still requires attribution ([see "Under the following terms" here](https://creativecommons.org/licenses/by/4.0/)).

You're removal here is good enough, however the binary data is still in the git history since you removed it in the latest commit, as opposed to an amended original commit, so the binary files are still present in the object history.